spyder/config/base.py

```diff
@@ -229,6 +229,7 @@ def get_image_path(name, default="not_found.png"):
 LANGUAGE_CODES = {'en': u'English',
                   'fr': u'Français',
                   'es': u'Español',
+                  'hu': u'Hungarian',

```

Comment:
Hungarian in English and Magyar in Hungarian :) I think fine both if either available